### PR TITLE
updating variables helper

### PR DESCRIPTION
**Pull request by iruzevic**
- updating css variables helper

### CODE REVIEW

**Review by volfkarlo — APPROVED** [review]
👏

File: src/Helpers/Components.php [hunk 1]
```diff
@@ -472,6 +484,18 @@ public static function camelToKebabCase(string $string): string
...
 	}
 
+	/**
+	 * Check if provided array is associative or sequential
+	 *
+	 * @param array $array Array to check.
+	 *
+	 * @return boolean
+	 */
+	public static function isAssoc(array $array): bool
```
Comment (dingo-d) [inline src/Helpers/Components.php:494]: Have you checked this implementation: https://wiki.php.net/rfc/is_list (the is_list will land in PHP 8.1 from what I see)?

I'm getting different results when I try their version and your helper

https://3v4l.org/M3vNs

**Review by goranalkovic-infinum — APPROVED** [review]

File: tests/Datasets/Arguments.php [hunk 2]
```diff
@@ -110,6 +110,20 @@
   ]]
 ]);
 
+dataset('isAssocWrong', [[
+	['a', 'b', 'c'],
+	['1', '2', '3'],
+	[],
+	'',
+	false,
+	['1' => 'a', '0' => 'b', '2' => 'c'],
```
Comment (IvanGrginovInf) [inline tests/Datasets/Arguments.php:119]: This one should still count as assoc array, no?

**Review by IvanGrginovInf — APPROVED** [review]
@iruzevic  Approved but left a comment regarding `isAssoc` test

**Review by dingo-d — CHANGES_REQUESTED** [review]
One test is wrong, that Grga mentioned, and I've left a comment about that polyfill, can you check it out?

You can also add the examples from the snippet to the test to see what will happen.

File: tests/Datasets/Arguments.php (hunk 2 above)
Comment (iruzevic) [reply tests/Datasets/Arguments.php:119]: yea this was switched now with the new code that Denis provided

File: src/Helpers/Components.php (hunk 1 above)
Comment (iruzevic) [reply src/Helpers/Components.php]: I have updated the new the function that you provided

**Comment by iruzevic** [comment]
@dingo-d not sure whey phpcs fails. I have run it locally and nothing reports

**Comment by dingo-d** [comment]
I am getting some errors, but also some false positives and I'm not sure why (can't reproduce them in the CS package).

The `{$e->getCode()}: {$e->getMessage()}` errors are legit, since you are printing something out, so you can escape those, but the two below (line 129 and 135 in the Components.php) are weird bug.

**Review by dingo-d — APPROVED** [review]

**Comment by iruzevic** [comment]
> I am getting some errors, but also some false positives and I'm not sure why (can't reproduce them in the CS package).
> 
> The `{$e->getCode()}: {$e->getMessage()}` errors are legit, since you are printing something out, so you can escape those, but the two below (line 129 and 135 in the Components.php) are weird bug.

I will test this and fix in the next pr

**Pull request merged**